pkg/controller/kfdef/kfdef_controller.go

```diff
@@ -245,11 +250,11 @@ var ownedResourcePredicates = predicate.Funcs{
 		if err != nil {
 			return false
 		}
-		log.Infof("Got update event for %v.%v.", object.GetName(), object.GetNamespace())
 		// if this object has an owner, let the owner handle the appropriate recovery
```

Comment:
Why did you delete this log info? I know we see a lot of them. but sometimes it helps to debug